openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -56,7 +56,7 @@ def _wrapped(*args, **kwargs):
                             func_name=func.func_name,
                             args=args,
                             kwargs=kwargs,
-                            exception=repr(ex)
+                            exception=ex.developer_message if hasattr(ex, 'developer_message') else repr(ex)
```

Comment:
This seems reasonable, but I don't see `developer_message` anywhere else in this PR. What things are raising exceptions with developer messages?